www/src/css/snaplist.css

```diff
@@ -21,31 +21,64 @@
 }
 
 .b-snaplist_grid .b-snaplist__item {
+  background-color:#fff;
+  margin-top:20px;
+  border-radius:10px;
+  box-shadow:0 1px 2px 0 rgba(0, 0, 0, 0.3);
+  overflow:hidden;
 }
 
-.b-snaplist_grid .b-snaplist__item * {
-  color:#999;
-  text-align: center;
+.b-snaplist_grid .b-snaplist__item:before {
+  content:'';
```

Comment:
Already commented about this in other PR, but seems it was introduced here.

This nested selector doesn't seem to be necessary. `b-snaplist__item` could work on its own.
Or does it need to have different styles when used inside and outside of grid?